pkg/component/runtime/service_command.go

```diff
@@ -55,24 +57,22 @@ func executeCommand(ctx context.Context, log *logger.Logger, binaryPath string,
 	// channel for the last error message from the stderr output
 	errch := make(chan string, 1)
 	ctxStderr := contextio.NewReader(ctx, proc.Stderr)
-	if ctxStderr != nil {
-		go func() {
-			var errText string
-			scanner := bufio.NewScanner(ctxStderr)
-			for scanner.Scan() {
-				line := scanner.Text()
-				if len(line) > 0 {
-					txt := strings.TrimSpace(line)
-					if len(txt) > 0 {
-						errText = txt
-						// Log error output line
-						log.Error(errText)
-					}
+	go func() {
+		var errText string
+		scanner := bufio.NewScanner(ctxStderr)
+		for scanner.Scan() {
+			line := scanner.Text()
+			if len(line) > 0 {
+				txt := strings.TrimSpace(line)
+				if len(txt) > 0 {
+					errText = txt
+					// Log error output line
+					log.Error(errText)
 				}
 			}
-			errch <- errText
-		}()
-	}
+		}
+		errch <- errText
+	}()
```

Comment:
This change is unrelated to the main changes in this PR.  It removes the `if ctxStderr != nil` conditional because the linter suggested that the condition would always evaluate to true.